samples/matter/lock/configuration/nrf5340dk_nrf5340_cpuapp/pm_static_dfu.yml

```diff
@@ -45,9 +45,49 @@ mcuboot_secondary_1:
     size: 0x40000
     device: MX25R64
     region: external_flash
-external_flash:
+app_1_download_app:
     address: 0x133000
-    size: 0x6CD000
+    size: 0xf3000
```

Comment:
I see that `0xf3000` is used for `mcuboot_secondary` slot, however regular app has size of `0xf2e00` is this correct?

Reply:
Yes, because mcuboot slot contains always app and additional 0x200 mcuboot padding.